src/Python/qsharp-core/qsharp/clients/iqsharp.py

```diff
@@ -168,12 +168,14 @@ def get_projects(self) -> List[str]:
         return self._execute("%project", raise_on_stderr=False)
 
     def simulate(self, op, **kwargs) -> Any:
+        kwargs.setdefault('timeout', None)
         return self._execute_callable_magic('simulate', op, **kwargs)
```

Comment:
I think the kwargs here get inserted into the command being executed (e.g.: `%simulate RunMain timeout=120`) rather than a parameter to `_execute` itself. We may need a new `_timeout_` parameter on `_execute_callable_magic` and `_execute`, similar to the existing `_quiet_` parameter.